Makefile.js

```diff
@@ -648,8 +669,20 @@ target.gensite = function(prereleaseVersion) {
             let text = cat(filename),
                 title;
 
-            // 5. Prepend page title and layout variables at the top of rules
+            // 6. Prepend page title and layout variables at the top of rules
             if (path.dirname(filename).indexOf("rules") >= 0) {
+
+                // Find out if the rule requires a special docs portion (e.g. if it is recommended and/or fixable)
+                const isRecommended = recommended.has(ruleName);
```

Comment:
@not-an-aardvark no worries, we all make mistakes sometimes.
@ilyavolodin will change as suggested, it does make the script a lot shorter!